bitnami/airflow/templates/deployment-scheduler.yaml

```diff
@@ -0,0 +1,177 @@
+          {{- else }}
+          value: {{ .Values.externalDatabase.user | quote }}
+          {{- end }}
+        - name: AIRFLOW_DATABASE_PASSWORD
+          valueFrom:
+            secretKeyRef:
+            {{- if .Values.postgresql.enabled }}
+              name: {{ template "airflow.postgresql.fullname" . }}
+              key: postgresql-password
+            {{- else }}
+              name: {{ printf "%s-%s" .Release.Name "externaldb" }}
+              key: db-password
+            {{- end }}
+        - name: AIRFLOW_DATABASE_HOST
+          {{- if .Values.postgresql.enabled }}
+          value: {{ template "airflow.postgresql.fullname" . }}
+          {{- else }}
+          value: {{ .Values.externalDatabase.host | quote }}
+          {{- end }}
+        - name: AIRFLOW_DATABASE_PORT_NUMBER
+          {{- if .Values.postgresql.enabled }}
+          value: "5432"
+          {{- else }}
+          value: {{ .Values.externalDatabase.port | quote }}
+          {{- end }}
+        - name: REDIS_HOST
+          {{- if .Values.redis.enabled }}
+          value: {{ template "airflow.redis.fullname" . }}
+          {{- else }}
+          value: {{ .Values.externalRedis.host | quote }}
+          {{- end }}
+        - name: REDIS_PORT_NUMBER
+          {{- if .Values.redis.enabled }}
+          value: "6379"
+          {{- else }}
+          value: {{ .Values.externalRedis.port | quote }}
+          {{- end }}
+        - name: REDIS_PASSWORD
+          valueFrom:
+            secretKeyRef:
+            {{- if .Values.redis.enabled }}
+              name: {{ template "airflow.redis.secretName" . }}
+              key: redis-password
+            {{- else }}
+              name: {{ printf "%s-%s" .Release.Name "externalredis" }}
+              key: redis-password
+            {{- end }}
+        - name: AIRFLOW_EXECUTOR
+          value: "CeleryExecutor"
+        - name: AIRFLOW_FERNET_KEY
+          valueFrom:
+            secretKeyRef:
+              name: {{ if .Values.airflow.auth.existingSecret }}{{ .Values.airflow.auth.existingSecret }}{{ else }}{{ template "airflow.fullname" . }}{{ end }}
+              key: airflow-fernetKey
+        - name: AIRFLOW_WEBSERVER_HOST
+          value: {{ template "airflow.fullname" . }}
+        - name: AIRFLOW_LOAD_EXAMPLES
+          {{- if .Values.airflow.loadExamples }}
+          value: "yes"
+          {{- else }}
+          value: "no"
+          {{- end }}
+        {{- if .Values.airflow.extraEnvVars }}
+          {{ toYaml .Values.airflow.extraEnvVars | nindent 8 }}
+        {{- end }}
+        {{- if .Values.livenessProbe.enabled }}
+        livenessProbe:
+        {{- end }}
+        {{- if .Values.readinessProbe.enabled }}
+        readinessProbe:
+        {{- end }}
+        volumeMounts:
+        {{- if or (.Files.Glob "files/dags/*.py") .Values.airflow.dagsConfigMap .Values.airflow.cloneDagFilesFromGit.enabled }}
+        - name: custom-dag-files
+          mountPath: /opt/bitnami/airflow/dags
```

Comment:
Can there be dags in a subfolder of dags? I see that you provide different ways to get the dag files, either using CMs or git repository. If we could have each option as subfolders, then both could be potentially compatible. If not, it is ok like this.

Reply:
I checked and it seems it can load DAGs from subfolders. I have made the changes so it is possible to load DAG files using the three options at the same time.